google/search/src/getMarkdown.ts

```diff
@@ -11,9 +11,7 @@ import * as cheerio from 'cheerio'
 export async function getMarkdown(page: Page, url: string): Promise<string> {
   try {
     await page.goto(url, { timeout: 1000 })
-  } catch (e) {
-    console.warn('slow page:', url)
```

Comment:
Looking for some guidance here. I removed these log statements to ensure the tool returned only JSON on the happy path. It was nice to know when a page was slow or couldn't be loaded, but it doesn't stop the tool from working.

Reply:
I assume `console.warn` prints to stdout. Does `console.error` work here? Iirc, gptscript does not send stderr to the LLM when the tool call is finished, only stdout.

Reply:
> I assume console.warn prints to stdout.

Actually, it's an alias for console.error, which already prints to stderr

>  Iirc, gptscript does not send stderr to the LLM when the tool call is finished, only stdout.

just double checked in otto and you're right. I must have misread the output when I initially tested the same query with `gptscript`, so these warn statements can stay.

Reply:
okay, updated